src/content/pages/sponsor.mdx

```diff
@@ -39,7 +39,7 @@ At EuroPython 2024, we offer eight main packages to keep EuroPython
 accessible to most organisations keen to help the community.
 
 <div style={{textAlign: "left", marginBottom: "30px"}}>
-<ButtonLink href="https://drive.google.com/file/d/1r-sGUI55BNRd0K48ypgvJ8QFucnaMdrK/view?usp=sharing">Sponsor Packages & Options (PDF)</ButtonLink>
+<ButtonLink href="https://drive.google.com/file/d/1KfqnSg-l1D5bUouj_jxMtqgugGpJ4rKQ/view">Sponsor Packages & Options (PDF)</ButtonLink>
 &nbsp;
 <ButtonLink href="https://forms.gle/ieK9LwdoRyDU4Z9a7">Sign up now!</ButtonLink>
```

Comment:
I am not sure this has been fixed but let's leave it as is and I will ask Konstantin.